src/lib.rs

```diff
@@ -623,33 +599,27 @@ impl<'input> Deref for Namespaces<'input> {
     }
 }
 
-
-#[derive(Clone, PartialEq)]
-struct ExpandedNameOwned<'input> {
-    ns: Option<CowStr<'input>>,
-    name: &'input str,
+#[derive(Copy, Clone)]
+struct NamespacedName<'input> {
```

Comment:
I think we should keep calling it `ExpandedNameSomething`, otherwise its a bit confusing.